banyand/metadata/schema/etcd_test.go

```diff
@@ -123,15 +124,19 @@ func useRandomTempDir() RegistryOption {
 	}
 }
 
-func useUnixDomain() RegistryOption {
+func useRandomPort() RegistryOption {
 	return func(config *etcdSchemaRegistryConfig) {
-		config.listenerClientURL, config.listenerPeerURL = randomUnixDomainListener()
+		ports, err := test.AllocateFreePorts(2)
+		if err != nil {
+			panic("fail to find free ports")
+		}
+		config.listenerClientURL, config.listenerPeerURL = fmt.Sprintf("http://127.0.0.1:%d", ports[0]), fmt.Sprintf("http://127.0.0.1:%d", ports[1])
```

Comment:
Why use `http` to replace `unix socket`? Do you tend to avoid creating local files?

Reply:
> Why use `http` to replace `unix socket`? Do you tend to avoid creating local files?

Yes

Reply:
Greate. it will help ppl run the server on a non-UNIX machine.